nginx/nginx.conf

```
@@ -1619,6 +1619,7 @@ http {
             rewrite "^/display/JENKINS/Terminology$" "https://www.jenkins.io/doc/book/glossary/" permanent;
             rewrite "^/display/JENKINS/Jenkins\+CLI$" "https://www.jenkins.io/doc/book/managing/cli/" permanent;
             rewrite "^/display/JENKINS/Jenkins\+SSH$" "https://www.jenkins.io/doc/book/managing/cli/" permanent;
+            rewrite "^/display/JENKINS/Dependencies+among+plugins" "https://www.jenkins.io/doc/developer/plugin-development/dependencies-and-class-loading/" permanent;
```

does this need escaping?
```suggestion
            rewrite "^/display/JENKINS/Dependencies\+among\+plugins" "https://www.jenkins.io/doc/developer/plugin-development/dependencies-and-class-loading/" permanent;
```